src/components/AQIGauge/AQIDial.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {Text, Box, Tag, Link, Center} from '@chakra-ui/react';
+import GaugeSVG from './GaugeSVG';
+import {useTranslation} from 'react-i18next';
+
+/**
+ * Interface for the props of the dial
+ * - `currentAQI` is the AQI value that the dial should display
+ */
+interface DialProps {
+  currentAQI: string;
+}
+
+/**
+ * AQI Dial Label Component
+ * This component displays the label for the AQI displayed by the dial.
+ * @example if the AQI is less than 50, it will show the label "Good"
+ */
+const AQILabel: ({currentAQI}: DialProps) => JSX.Element = ({
```

Comment:
I think it's more typical to have only the first letter of the acronyms capitalized in code, like `UrlParser`

```suggestion
const AqiLabel: ({currentAqi}: DialProps) => JSX.Element = ({
```